iis-wf/iis-wf-export-actionmanager/src/main/java/eu/dnetlib/iis/wf/export/actionmanager/api/ActionManagerServiceFacade.java

```diff
@@ -20,19 +22,8 @@
 	 * @param provenance
 	 * @param trust
 	 * @param nsprefix
-	 * @return number ofactions successfully stored
 	 * @throws ActionManagerException
```

Comment:
It's good you pointed that out. After CNR refactored `ActionManager` model some time ago it turns out we can finally simplify `ActionManagerServiceFacade` API by removing most of those parameters.